sympy/geometry/ellipse.py

```diff
@@ -543,6 +543,33 @@ def focus_distance(self):
         """
         return Point.distance(self.center, self.foci[0])
 
+    def auxiliary_circle(self, x='x', y='y'):
+         """The equation of auxiliary circle of the ellipse.
+
+         Returns
+         =======
+
+         equation : sympy expression
+
+         Parameters
+         ==========
+
+         x : str, optional Label for the x-axis. Default value is 'x'.
+         y : str, optional Label for the y-axis. Default value is 'y'.
+
+         Examples
+         ========
+
+         >>> from sympy import Ellipse, Point
+         >>> ellipse = Ellipse(Point(2, 4), 9, 1)
+         >>> ellipse.auxiliary_circle()
+         (x - 2)**2 + (y - 4)**2 - 81
+
+        """
```

Comment:
I think every extra spcing ' ' above this like is causing the indentation error.
Check if you have any line indented more than 8 spaces.

Reply:
Probably you should revert 6f3a72bb609809e89e666dfad4da5d9a469c6936

Reply:
Probably you should revert 6f3a72bb609809e89e666dfad4da5d9a469c6936